<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号与槽的使用</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/res.qrc">
    <normaloff>:/icons/images/editor.ico</normaloff>:/icons/images/editor.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_color">
       <property name="title">
        <string>字体</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cb_under">
          <property name="text">
           <string>Underline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_italic">
          <property name="text">
           <string>Italic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_bold">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_font">
     <property name="title">
      <string>颜色</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtn_black">
        <property name="text">
         <string>Black</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_red">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_blue">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pt_edit_display">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>Hello World!
你好，世界！</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbtn_clear">
       <property name="text">
        <string>清空</string>
       </property>
       <property name="icon">
        <iconset resource="Resources/res.qrc">
         <normaloff>:/icons/images/212.bmp</normaloff>:/icons/images/212.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_confirm">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="Resources/res.qrc">
         <normaloff>:/icons/images/322.bmp</normaloff>:/icons/images/322.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_quit">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="Resources/res.qrc">
         <normaloff>:/icons/images/324.bmp</normaloff>:/icons/images/324.bmp</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbtn_quit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbtn_confirm</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
